### ISSUE TITLE
lr_rate

### ISSUE THREAD

**Issue by ydhongHIT** [HiKapok/SSD.TensorFlow#110]
At the first 500 steps, the learning rate is only one-tenth of the initial learning rate. After that, learning rate is ten times bigger and the same as the initial value. Why?
